hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -85,28 +95,13 @@ const Orders = () => {
                                         key={idx}
                                         onClick={() => handleClick(order.id)}
```

Comment:
Can you replace the onClick function with history.push(`/teams/${order.team_code}`)

Reply:
done